.github/workflows/build.yaml

```diff
@@ -0,0 +1,40 @@
+name: Build
+on: [push, pull_request]
+jobs:
+  build_docker:
+    name: Build Docker Image
+    runs-on: ubuntu-20.04
+    env:
+      DOCKER_TAG: cabforum/build-guidelines-action
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Build Image
+        run: docker build --tag ${{ env.DOCKER_TAG}} .
+      - name: Save image
+        run: |
+          docker save ${{ env.DOCKER_TAG }} |\
+          gzip > build-guidelines-action.tgz
+      - name: Upload archive
+        uses: actions/upload-artifact@v1
```

Comment:
Both the upload-artifact and download-artifact actions have a stable v2 release. Might as well use that? (Although note that moving to v2 makes the comment below about setting the `path` a bit less optional, as the default download location of download-artifact seems to be one of the only real changes between v1 and v2.)